tools/pythonpkg/tests/fast/spark/test_spark_dataframe.py

```diff
@@ -47,22 +58,29 @@ def test_dataframe_from_list_of_tuples(self, spark):
             df = spark.createDataFrame(5, ["id", "address", "test"])
 
         # Empty list
-        df = spark.createDataFrame([], ["id", "address", "test"])
-        res = df.collect()
-        assert res == []
+        if not USE_ACTUAL_SPARK:
+            # Spark raises PySparkValueError [CANNOT_INFER_EMPTY_SCHEMA]
```

Comment:
Can you preface this with `FIXME: `
I think we want to fix this in the future, again something we would never catch without testing against the real spark :pray: